src/profile/utils.js

```diff
@@ -104,13 +141,15 @@ function createCard({ oldData, newData, userId, username, profileDiffId }) {
     parent: oldDataContainer,
   });
 
+  const diffValues = checkDifferentValues(oldData, newData);
+
   const oldUserInfoList = createCardComponent({
     tagName: 'ul',
     className: 'userInfoListContainer',
     parent: oldDataContainer,
   });
 
-  displayList(oldData, oldUserInfoList);
+  displayList(oldData, oldUserInfoList, diffValues, OLD_DATA);
```

Comment:
NIT: Pass parameters in an object like displayList(options);

Reply:
Thank you @rajakvk 
@vin18 let's consider while refactoring

Reply:
cc @rohan09-raj

Reply:
Got it, @Pavangbhat. Will handle this in global refactor PR. 
Thank you @rajakvk for the review.